cwms-data-api/src/test/resources/cwms/cda/data/dto/timeseriesprofile/timeseriesprofile.json

```diff
@@ -0,0 +1,8 @@
+{
+  "description": "Description",
+  "parameter-list": ["Temperature","Depth"],
+  "location-id": "Location",
+  "office-id": "Office",
+  "ref-ts-id": "TimeSeries",
```

Comment:
Use a correctly formatted time series id for examples

Reply:
the position of the parameter refers to the location of the timeseries of the specified parameter inside an input data block. In this format without the datablock the position is irrelevant.

Reply:
The position is not exposed by the API and is only used for DB parameter ordering. We do not need the int value outside of the database.